### PR TITLE
Add ability to disable form control via config.xml

### PR DESCRIPTION
**Pull request by bryanhiggins**
https://github.com/blackberry/BB10-WebWorks-Framework/issues/391

### CODE REVIEW

**Comment by nukulb** [comment]
needs microsite issue

**Comment by nukulb** [comment]
@tracyli - needs testing

**Comment by bryanhiggins** [comment]
http://github.rim.net/Microsites/HTML5-WebWorks/issues/423

**Comment by tracyli** [comment]
tested with the related framework pull request and please refer to https://github.com/blackberry-webworks/BB10-WebWorks-Framework/pull/294 for result.

File: lib/config-parser.js [hunk 1]
```diff
@@ -614,6 +615,10 @@ function init() {
                     widgetConfig.enableChildWebView = ((childBrowser + '').toLowerCase() === 'disable') === false;
                 }
 
+                if (formControl) {
+                    widgetConfig.enableFormControl = ((formControl + '').toLowerCase() === 'disable') === false;
```
Comment (jeffheifetz) [inline lib/config-parser.js:619]: @jkeshavarzi @nukulb I'm making a similar change for websecurity and want to know if I should be doing this too. Should we be allowing case insensitivity or booleans without a string ?

Comment (nukulb) [inline lib/config-parser.js:619]: ummm, I wish had a clear answer
w3c spec doesn't specify
but I think its a goog feature

so I am saying case insensitive is a better but we should probably open an issue for the other features that don't do that right now.

**Comment by nukulb** [comment]
merged manually

File: lib/config-parser.js (hunk 1 above)
Comment (jeffheifetz) [inline lib/config-parser.js:619]: Opened blackberry/BB10-Webworks-Packager#224 to track

**Pull request closed without merging**